### ISSUE TITLE
Incorrect generated code for multiple exceptions

### ISSUE THREAD

**Issue by treecy** [creditkarma/thrift-typescript#129]
Given Thrift:
```thrift
service DruidQueryService {
  QueryResult query(
    1: required string       source,
  ) throws (
    1: ColumnNotExistsException      columnNotExists,
    2: DonatelloException            donatelloException,
  ),
}
```
The generated code:
```typescript
public process_query(requestId: number, input: thrift.TProtocol, output: thrift.TProtocol): void {
  new Promise<QueryResult>((resolve, reject): void => { ... })
    .then()
    .catch((err: Error): void => {
      if (err instanceof ColumnNotExistsException) {
        const result: QueryResult = new QueryResult({ columnNotExists: err });
        output.writeMessageBegin("query", thrift.Thrift.MessageType.REPLY, requestId);
        result.write(output);
        output.writeMessageEnd();
        output.flush();
        return;
      }
      else {
        const result: thrift.Thrift.TApplicationException = new thrift.Thrift.TApplicationException(thrift.Thrift.TApplicationExceptionType.UNKNOWN, err.message);
        output.writeMessageBegin("query", thrift.Thrift.MessageType.EXCEPTION, requestId);
        result.write(output);
        output.writeMessageEnd();
        output.flush();
        return;
      }

      // ↓↓↓↓↓↓↓ The following part did get run ↓↓↓↓↓↓↓
      if (err instanceof DonatelloException) {
        const result: QueryResult = new QueryResult({ donatelloException: err });
        output.writeMessageBegin("query", thrift.Thrift.MessageType.REPLY, requestId);
        result.write(output);
        output.writeMessageEnd();
        output.flush();
        return;
      }
      else {
        const result: thrift.Thrift.TApplicationException = new thrift.Thrift.TApplicationException(thrift.Thrift.TApplicationExceptionType.UNKNOWN, err.message);
        output.writeMessageBegin("query", thrift.Thrift.MessageType.EXCEPTION, requestId);
        result.write(output);
        output.writeMessageEnd();
        output.flush();
        return;
      }
    })
}
```

The code should be like:
```typescript
public process_query(requestId: number, input: thrift.TProtocol, output: thrift.TProtocol): void {
  new Promise<QueryResult>((resolve, reject): void => { ... })
    .then()
    .catch((err: Error): void => {
      if (err instanceof ColumnNotExistsException || err instanceof DonatelloException) { // <-----FIXED
        const result: QueryResult = new QueryResult({ columnNotExists: err });
        output.writeMessageBegin("query", thrift.Thrift.MessageType.REPLY, requestId);
        result.write(output);
        output.writeMessageEnd();
        output.flush();
        return;
      }
      else {
        const result: thrift.Thrift.TApplicationException = new thrift.Thrift.TApplicationException(thrift.Thrift.TApplicationExceptionType.UNKNOWN, err.message);
        output.writeMessageBegin("query", thrift.Thrift.MessageType.EXCEPTION, requestId);
        result.write(output);
        output.writeMessageEnd();
        output.flush();
        return;
      }
}
```

**Comment by kevin-greene-ck** [comment]
Thanks for the detailed report. I should be able to get this fixed today.

**Comment by kevin-greene-ck** [comment]
Released in 2.0.8
